.gitignore

```diff
@@ -2,13 +2,18 @@
 *.egg-info
 *.eggs
 
+# Editor environment
```

Comment:
Since different people use different editors, it's generally better to put ignore rules for editor files in a global configuration file. By default on Linux, that's `~/.config/git/ignore`. So please remove this (and yes, that `*.swp` below should also go then).